WordPressEditor/WordPressEditor/Classes/Plugins/WordPressPlugin/Gutenberg/GutenpackConverter.swift

```diff
@@ -0,0 +1,35 @@
+import Aztec
+import Foundation
+
+public extension Element {
+    static let gutenpack = Element("gutenpack")
+}
+
+public class GutenpackConverter: ElementConverter {
+    
+    // MARK: - ElementConverter
+    
+    public func convert(
+        _ element: ElementNode,
+        inheriting attributes: [NSAttributedStringKey: Any],
+        childrenSerializer serializeChildren: ChildrenSerializer) -> NSAttributedString {
+        
+        precondition(element.type == .gutenpack)
+        
+        let attachment = HTMLAttachment()
+        let decoder = GutenbergAttributeDecoder()
+        if let content = decoder.decodedAttribute(named: GutenbergAttributeNames.selfCloser, from: element) {
+            attachment.rawHTML = String(content.prefix(upTo: content.index(before:content.endIndex)))
```

Comment:
This line if fine, but just a suggestion because I love the syntax of subscripts:

```swift
attachment.rawHTML = content[content.startIndex ..< content.index(before: content.endIndex)]
```

Feel free to disregard.